corehq/warehouse/management/commands/clear_staging_records.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management import BaseCommand
+from corehq.warehouse.const import STAGING_TABLES
+from corehq.warehouse.models import get_cls_by_slug
+
+
+USAGE = """Usage: ./manage.py clear_staging_records
+
+Removes all records from staging tables
```

Comment:
will this be called by the warehouse load process? it might be better to clear one staging table at a time right before loading the respective dim/fact, in case we ever need to restart any individual jobs